tests/spiffe/README.txt

```diff
@@ -0,0 +1,54 @@
+An SVID is a structured x509 certificate that follows the SPIFFE x509-SVID
```

Comment:
One last nit: I think we want this file to be `README.md` instead of `README.txt`.